roles/accprovision/manifests/net-operator-controller-deployment.yaml

```diff
@@ -0,0 +1,464 @@
+                        type: object
+                        properties:
+                          as_number:
+                            type: integer
+                          racks:
+                            type: array
+                            items:
+                              type: array
+                              items:
+                               type: string
+                  profile:
+                    type: string
+            type: object
+          status:
+            description: AccProvisionInputStatus defines the successful completion of AccProvisionInput
+            properties:
+              status:
+                type: boolean
+            type: object
+        required:
+        - spec
+        type: object
+---
+apiVersion: v1
+data:
+  spec: "{\n    \"acc_provision_input\": {\n        \"operator_managed_config\": {\n\
+    \            \"enable_updates\": false\n        },\n        \"aci_config\": {\n\
+    \            \"system_id\": \"openupi\",\n            \"apic_hosts\": [\n    \
+    \            \"10.30.120.120\"\n            ],\n            \"aep\": \"sauto_k8s-bm-1_aep\"\
+    ,\n            \"tenant\": {\n                \"name\": \"prj_5cc3c29ec36a49cf942ff84f9e8deb30\"\
+    \n            },\n            \"vrf\": {\n                \"name\": \"sauto_k8s-bm-1_l3out-1_vrf\"\
+    ,\n                \"tenant\": \"common\"\n            },\n            \"l3out\"\
+    : {\n                \"name\": \"sauto_k8s-bm-1_l3out-1\",\n                \"\
+    external_networks\": [\n                    \"sauto_k8s-bm-1_l3out-1_epg\"\n \
+    \               ]\n            }\n        },\n        \"registry\": {\n      \
+    \      \"image_prefix\": \"quay.io/noirolabs\", \n            \"image_pull_secret\"\
+    : \"noiro-docker-registry-secret\", \n            \"aci_containers_controller_version\"\
+    : \"5.2.3.1.1d150da\", \n            \"aci_containers_host_version\": \"5.2.3.1.1d150da\"\
+    , \n            \"cnideploy_version\": \"5.2.3.1.1d150da\", \n            \"opflex_agent_version\"\
+    : \"5.2.3.1.1d150da\", \n            \"openvswitch_version\": \"5.2.3.1.1d150da\"\
+    , \n            \"aci_containers_operator_version\": \"5.2.3.1.1d150da\"\n   \
+    \     },\n        \"logging\": {\n           \"controller_log_level\": \"debug\"\
+    , \n           \"hostagent_log_level\": \"debug\", \n           \"opflexagent_log_level\"\
+    : \"trace\"\n        },\n        \"net_config\": {\n            \"infra_vlan\"\
+    : 4093,\n            \"service_vlan\": 2033, \n            \"extern_static\":\
+    \ \"151.10.1.1/21\",\n            \"extern_dynamic\": \"151.9.1.1/24\",\n    \
+    \        \"node_svc_subnet\": \"15.5.168.1/21\",\n            \"interface_mtu\"\
+    : 1400,\n            \"pod_subnet_chunk_size\": 24,\n            \"node_subnet\"\
+    : \"15.11.0.1/27\",\n            \"pod_subnet\": \"15.128.0.1/16\"\n        }\n\
+    \    }\n }"
```

Comment:
i think this spec string should be different, in fact it should ideally be empty to begin with...

Reply:
Checking....

Reply:
Checked. Currently needed the pod_subnet and rest other fields in net_config. Otherwise, it says missing option. We need to support the downstream acc-provision to remove the rest of the irrelevant fields.

Reply:
HI @abhis2112, I have removed the "node_svc_subnet", "pod_subnet_chunk_size", "extern_static" and "svc_vlan" fields from the required options in my acc-provision branch. When using "cko-calico" flavor, these fields are no longer required and can be safely removed.